src/components/ThumbnailImage.js

```diff
@@ -87,9 +86,7 @@ class ThumbnailImage extends PureComponent {
     }
 
     render() {
-        const url = this.props.isAuthTokenRequired
-            ? addEncryptedAuthTokenToURL(this.props.previewSourceURL)
-            : this.props.previewSourceURL;
+        const url = this.props.previewSourceURL;
```

Comment:
This is only used at one place, I think we can just use `this.props.previewSourceURL`.